client/my-sites/earn/memberships/index.jsx

```diff
@@ -335,20 +361,36 @@ class MembershipsSection extends Component {
 				>
 					<h1>{ this.props.translate( 'Confirmation' ) }</h1>
 					<p>
-						{ this.props.translate(
-							'Do you want to disconnect Recurring Payments from your Stripe account?'
-						) }
+						{ isPayments
+							? this.props.translate(
+									'Do you want to disconnect Payments from your Stripe account?'
+							  )
+							: this.props.translate(
+									'Do you want to disconnect Recurring Payments from your Stripe account?'
+							  ) }
 					</p>
 					<Notice
-						text={ this.props.translate(
-							'Once you disconnect Recurring Payments from Stripe, new subscribers won’t be able to sign up and existing subscriptions will stop working.{{br/}}{{strong}}Disconnecting your Stripe account here will remove it from all your WordPress.com and Jetpack sites.{{/strong}}',
-							{
-								components: {
-									br: <br />,
-									strong: <strong />,
-								},
-							}
-						) }
+						text={
+							isPayments
+								? this.props.translate(
+										'Once you disconnect Payments from Stripe, new subscribers won’t be able to sign up and existing subscriptions will stop working.{{br/}}{{strong}}Disconnecting your Stripe account here will remove it from all your WordPress.com and Jetpack sites.{{/strong}}',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Once you disconnect Recurring Payments from Stripe, new subscribers won’t be able to sign up and existing subscriptions will stop working.{{br/}}{{strong}}Disconnecting your Stripe account here will remove it from all your WordPress.com and Jetpack sites.{{/strong}}' )` *ES Score: 11*